### DISCUSSION TITLE
Slower since installing

### DISCUSSION THREAD

**Discussion by rendez2k** [Guilouz/Klipper-Flsun-Speeder-Pad#144] [General]
Hi all, since installing, the printer is visibly, audibly and (via the print times), so much slower than it was! Is there a setting somewhere in mainsail? Its at 100% speed currently. Before install, I could hear it rattling away downstairs!

**Comment by kaktuss77** [comment]
Hi, I confirm, mine is slower too, but quality of printing is better.
For exemple I do infill @400mm/s now it cannot be reached. (same file as before).
Velocity is set to 600mm/s
acceleration is set to 10k

I don't understand why. I have backup the printer.cfg before migrating to official klipper, I will compare the 2 files.

Regards

**Reply by kaktuss77** [reply]
I was Wrong, just set Jerk setting to Orca to up square corner velocity for infill parterns.

![image](https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad/assets/76732050/26d6cdb5-7de7-415d-a411-2baa9b58803f)

don't forget to change max limit for jerk control into printer setting (Orca Slicer / Prusa Slicer)

Regards

```
Maybe I find why, 

OEM v1.2 Speeder Pad printer.cfg file printer section 
![image](https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad/assets/76732050/fb311b0d-d201-4a82-b49b-95361a58ff55)

printer.cfg on Github
![image](https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad/assets/76732050/5485a9f2-66ca-4883-b432-3195b38c10ed)

I'm printing right now, will test after
```
